x-pack/solutions/search/plugins/enterprise_search/public/applications/enterprise_search_content/components/connectors/create_connector/finish_up_step.tsx

```diff
@@ -311,9 +310,7 @@ export const FinishUpStep: React.FC<FinishUpStepProps> = ({ title }) => {
                         'Use your favorite language client to query your data in your app',
                     }
                   )}
-                  onClick={() => {
-                    application.navigateToUrl(http.basePath.prepend(ELASTICSEARCH_PLUGIN.URL));
-                  }}
+                  href={docLinks.languageClients}
```

Comment:
I was a bit unsure on this one, @sphilipse or @seanstory any opinions on better places to send users for this link with the Elasticsearch getting started guide being removed?

Reply:
🤷 seems reasonable to me.